gensim/parsing/preprocessing.py

```diff
@@ -48,12 +68,36 @@ def remove_stopwords(s):
 
 
 def strip_punctuation(s):
+    """Takes string, replaces all punctuation characters with spaces.
+
+    Parameters
+    ----------
+    s : str
+
+    Returns
+    -------
+    str
+        Unicode string without punctuation characters.
+
+    Examples
+    --------
+    >>> from gensim.parsing.preprocessing import strip_punctuation
+    >>> s = "A semicolon is a stronger break than a comma, but not as much as a full stop!"
+    >>> strip_punctuation(s)
+    u'A semicolon is a stronger break than a comma  but not as much as a full stop '
+
+    """
+
     s = utils.to_unicode(s)
     return RE_PUNCT.sub(" ", s)
 
 
 # unicode.translate cannot delete characters like str can
 strip_punctuation2 = strip_punctuation
+"""
```

Comment:
That won't work, this is not how docstrings work.

Reply:
No need docstring here, this will be removed in refactoring.